### ISSUE TITLE
Add possibility to generate signed commits

### ISSUE THREAD

**Issue by Viserion77** [githubnext/copilot-workspace-user-manual#65]
I tested the tool for the first time today and I found it incredible and very practical! I just have to thank everyone involved!

Well, in this test I realized that the tool generated a commit with my name, but without any signature.

Would it be possible to integrate it with Github to generate the signature automatically? (This already happens in codespaces, for example)

Now:
![image](https://github.com/user-attachments/assets/43446ef4-d452-4392-acd7-dc5857f2a08c)
Expect:
![image](https://github.com/user-attachments/assets/e3dbc9b4-33dc-4da2-a9f4-cbaf7e573e6b)

**Comment by obryckim** [comment]
Agree; we have a requirement for all commits to be signed. It would be helpful to have the tool sign the commits just as if making the changes in github.com, etc.

**Comment by neilenns** [comment]
I just hit this as well, I have signed commits required on my repo and was messing around with copilot workspaces to generate unit tests. I had copilot create a PR, then I made some additional changes. My commit is verified but the original copilot one isn't, and github won't let me merge.

![image](https://github.com/user-attachments/assets/bc7aa033-c9a9-4d09-8670-f8291362ead5)
